packages/expo-live-photo/src/LivePhotoView.tsx

```diff
@@ -0,0 +1,63 @@
+import { Platform, requireNativeViewManager, UnavailabilityError } from 'expo-modules-core';
+import * as React from 'react';
```

Comment:
```suggestion
import { ComponentType, etc. } from 'react';
```

Let's use named imports for better docs readability.